internal/handlers/handler_test.go

```diff
@@ -591,7 +662,30 @@ func TestFind(t *testing.T) {
 					}
 
 					actual := handle(ctx, t, handler, tc.req)
-					assert.Equal(t, expected, actual)
+					if tc.deep {
```

Comment:
That looks like a workaround for some problem. What is it?

Reply:
It doesn't work as expected: Document m field is private and is not being marshaled.